Formula/hashlink.rb

```diff
@@ -0,0 +1,32 @@
+class Hashlink < Formula
+  desc "Virtual machine for Haxe"
+  homepage "https://hashlink.haxe.org/"
+  url "https://github.com/HaxeFoundation/hashlink.git",
+    :revision => "cd269136628a1d445b2253ded9fd10ab9fe2f9be",
+    :tag      => "1.11"
+
+  depends_on "cmake"
```

Comment:
Is this a runtime dependency?

Reply:
The short answer is no.  These are (mostly) all lazy loaded for the purposes of FFI.